### PR TITLE
Create method

### PR DESCRIPTION
**Pull request by zxqfox**
cc @blond @tadatuta @yeti-or

Let's do this without major changes first because it breaks bem-cell and the rest

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -285,4 +285,50 @@ module.exports = class BemEntityName {
     static isBemEntityName(entityName) {
         return entityName && entityName.__isBemEntityName__;
     }
+
+    /**
+     * Creates BemEntityName instance by the any object representation.
```
Comment (tadatuta) [inline index.js]: ~~the~~

File: index.js [hunk 2]
```diff
@@ -285,4 +285,50 @@ module.exports = class BemEntityName {
...
+     *
+     * @param {object} obj — representation of entity name.
+     * @param {string} obj.block  — the block name of entity.
+     * @param {string} [obj.elem] — the element name of entity.
+     * @param {object|string} [obj.mod]  — the modifier of entity.
+     * @param {string} [obj.mod.name] — the modifier name of entity.
+     * @param {string} [obj.mod.val]  — the modifier value of entity.
+     * @param {string} [obj.modName] — the modifier name of entity.
```
Comment (tadatuta) [inline index.js]: let's add notice about priority?

File: README.md [hunk 3]
```diff
@@ -255,6 +241,57 @@ name.valueOf();
...
+
+BemEntityName.isBemEntityName(entityName); // true
+BemEntityName.isBemEntityName({}); // false
+```
+
+### #create(object)
+
+Creates BemEntityName instance by any object representation.
+
+Helper for laziness.
```
Comment (zxqfox) [inline README.md]: Ok: `Helper for sugar-free simplicity.`

File: README.md [hunk 4]
```diff
@@ -255,6 +239,57 @@ name.valueOf();
...
+-------------|-----------------|-----------------------
+`entityName` | `BemEntityName` | The entity to check.
+
+```js
+const BemEntityName = require('@bem/entity-name');
+
+const entityName = new BemEntityName({ block: 'input' });
+
+BemEntityName.isBemEntityName(entityName); // true
+BemEntityName.isBemEntityName({}); // false
```
Comment (Yeti-or) [inline README.md]: `BemEntityName.isBemEntityName({ block: 'button' }); // false

File: README.md [hunk 5]
```diff
@@ -255,6 +239,57 @@ name.valueOf();
...
+
+Creates BemEntityName instance by any object representation.
+
+Helper for sugar-free simplicity.
+
+Parameter    | Type               | Description
+-------------|--------------------|--------------------------
+`object`     | `object`           | Representation of entity name.
+
+Passed Object can have the common field names for entities:
```
Comment (Yeti-or) [inline README.md]: could have ?

**Pull request merged**